modules/images-configuration-parameters-hcp.adoc

```diff
@@ -0,0 +1,44 @@
+====
+
+[cols="3a,8a",options="header"]
+|===
+|Parameters for {product-rosa} CLI |Description
+
+|`registry-config-allowed-registries`
+|Registries for which image pull and push actions are allowed. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are blocked.
+
+|`registry-config-insecure-registries`
+|Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
+
+The namespace for this config map is `openshift-config`. The format of the config map is to use the registry hostname as the key, and the PEM-encoded certificate as the value, for each additional registry CA to trust.
+
+|`registry-config-blocked-registries`
+|Registries for which image pull and push actions are denied. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are allowed.
+
+|`registry-config-allowed-registries-for-import`
+|Contains configuration that determines how the container runtime should treat individual registries when accessing images for builds and pods. For example, whether or not to allow insecure access. It does not contain configuration for the internal cluster registry.
+
+|`registry-config-platform-allowlist`
+|Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
+
+|`registry-config-additional-trusted-ca`
+| A json file containing the registry hostname as the key, and the PEM-encoded certificate as the value, for each additional registry CA to trust.
+
+|===
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, all registries, including the `registry.redhat.io` and `quay.io` registries and the default {product-registry}, are blocked unless explicitly listed. If you use the parameter, to prevent pod failure, add the `registry.redhat.io` and `quay.io` registries and the `internalRegistryHostname` to the `allowedRegistries` list, as they are required by payload images within your environment. For disconnected clusters, mirror registries should also be added.
```

Comment:
The Red Hat list is automatically handled. I would change this to something like this.
```suggestion
When the `allowedRegistries` parameter is defined, all registries, are blocked unless explicitly listed. To prevent pod failure, a list of Red Hat registries is automatically whitelisted, as they are required by payload images within your environment. For disconnected clusters, mirror registries should be added by the user itself.
```

Reply:
updated